.cr.yaml

```diff
@@ -0,0 +1,4 @@
+# This file is the config file for helm/chart-releaser
+owner: topolvm
+git-repo: topolvm
+release-name-template: {{ .Name }}-chart-v{{ .Version }}
```

Comment:
The release name of topolvm chart will be `topolvm-chart-v1.2.3`.
A git tag will be pushed with the same name.

Reply:
I think it is no problem.